src/conductor/client/worker/worker_interface.py

```diff
@@ -42,7 +44,16 @@ def get_task_definition_name(self) -> str:
 
         :return: TaskResult
         """
-        return self.task_definition_name
+        if isinstance(self.task_definition_names, list):
+            if self.next_task_index >= len(self.task_definition_names):
+                self.next_task_index = 0
+            task_definition_name = self.task_definition_names[self.next_task_index]
+            if self.next_task_index == (len(self.task_definition_names) - 1):
```

Comment:
Basic round robin between tasks in the Worker

Reply:
Couldn't we just do modulo - ` self.next_task_index = (self.next_task_index + 1) %  len(self.task_definition_names)` ?

Reply:
I think you are right -- all set.